source/classes.tex

```diff
@@ -1879,40 +1879,20 @@
 
 \begin{itemize}
 \item
-each direct base class
-\tcode{B}
-of
-\tcode{X}
+each direct base class \tcode{B} of \tcode{X}
 has a copy assignment operator whose parameter is of type
-\tcode{const}
-\tcode{B\&},
-\tcode{const}
-\tcode{volatile}
-\tcode{B\&}
-or
-\tcode{B},
-and
+\tcode{const B\&}, \tcode{const volatile B\&}, or \tcode{B}, and
 \item
-for all the non-static data members of
-\tcode{X}
-that are of a class type
-\tcode{M}
-(or array thereof),
+for all the non-static data members of \tcode{X} that are of a class type
+\tcode{M} (or array thereof),
```

Comment:
Since you're re-flowing the source here: We want [semantic linebreaks](http://www.sembr.org), but the best position to break seems to be before the "that" (which starts a relative clause).